src/NumericalAlgorithms/Interpolation/CMakeLists.txt

```diff
@@ -32,5 +32,6 @@ target_link_libraries(
   Blas
   Domain
   ErrorHandling
+  GSL::gsl
```

Comment:
what made this target?

Reply:
CMake's packaged `FindGSL` provides this, at least since CMake 3.3 (I didn't check earlier since that's all we require). Similarly, `FindBoost` exposes `Boost::boost`, etc.